src/entities/Bullet_test.js

```diff
@@ -52,3 +53,48 @@ describe('Bullet', () => {
     });
   });
 });
+
+describe('PlasmaBullet', () => {
+  const position = new Vec2(2, 2);
+  const direction = new Vec2(0, 1);
+
+  it('is orange', () => {
+    const bullet = new PlasmaBullet(position, direction, null);
+    assert(bullet.color == 'orange');
```

Comment:
Use `assert.equal`. You'll get a better error message.

`"red" doesn't equals "orange"` instead of  just `assertion failed`.